www/src/pages/guidelines/logo.js

```diff
@@ -0,0 +1,658 @@
+              Blue
+            </Text>
+            {` `}
+            marks type and element alignment and construction.
+          </Text>
+        </CopyColumn>
+        <ContentColumn>
+          <Box
+            mb={4}
+            dangerouslySetInnerHTML={{
+              __html: Clearspace,
+            }}
+            css={{ svg: { display: `block`, maxWidth: 506 } }}
+          />
+          <Text as="p" mb={7} fontSize={1} color="grey.50">
+            <Text as="span" color="magenta.50">
+              Clearspace
+            </Text>
+            {` `}
+            around the logo is equal to the wordmark x-height.
+          </Text>
+          <Box
+            mb={4}
+            dangerouslySetInnerHTML={{
+              __html: ClearspaceMonogram,
+            }}
+            css={{ svg: { display: `block`, maxWidth: 122 } }}
+          />
+          <Text as="p" mb={0} fontSize={1} color="grey.50">
+            <Text as="span" color="magenta.50">
+              Clearspace
+            </Text>
+            {` `}
+            around the monogram equals 1/4 of its height.
+          </Text>
+        </ContentColumn>
+      </Columns>
+      <SectionHeading id="scale">Scale</SectionHeading>
+      <Columns>
+        <CopyColumn sticky={false}>
+          <p>
+            Our logo is designed to scale to small sizes on print and screen.
+          </p>
+          <p>
+            Smallest size: 24 pixels high for screens, 0.3 inch/0.762 centimeter
+            high for print.
+          </p>
+        </CopyColumn>
+        <ContentColumn>
+          <div css={{ position: `relative` }}>
+            <GatsbyLogoContainered opacity={0.025} />
+            <div
+              css={{
+                position: `absolute`,
+                zIndex: 1,
+                height: 24,
+                top: `auto`,
+                bottom: 0,
+              }}
+            >
+              <GatsbyLogoContainered height="24px" />
+            </div>
+          </div>
+          <Text as="p" mt={2} fontSize={1} color="grey.50">
+            Logo at 24px height
+          </Text>
+        </ContentColumn>
+      </Columns>
+    </Section>
+
+    <Section id="guidance">
+      <SectionHeading>Guidance</SectionHeading>
+      <Columns>
+        <CopyColumn>
+          <p>
+            Please help us maintain the integrity of the Gatsby logo and promote
+            the consistency of the brand by not misusing it. Be responsible, not
+            reckless.{` `}
+            <span role="img" aria-label="thank you">
+              🙏
+            </span>
+          </p>
+          <p>
+            If you find your needs are not covered by the logo and its
+            recommended usage, please{` `}
+            <a href="mailto:team@gatsbyjs.com">get in touch</a>.
+          </p>
+        </CopyColumn>
+        <ContentColumn>
+          <Flex flexWrap="wrap">
+            {data.allGuidanceYaml.edges.map(({ node }, index) => (
+              <Guidance
```

Comment:
Can these items be rendered in an unordered list to add some semantics? A `<ul>` wrapper could be added outside of the loop and each `<Guidance>` item could either be wrapped in `<li>` or the component itself could be updated.

Reply:
Fixed in 34aa733